packages/components/form/src/form.ts

```diff
@@ -126,6 +136,19 @@ export class Form<T extends Record<string, unknown> = Record<string, unknown>> e
     return this.controls.map(c => c.reportValidity()).every(Boolean);
   }
 
+  reset(): void {
```

Comment:
Can you please add the description that will be shown on the documentation website as well?

<img width="923" alt="image" src="https://github.com/user-attachments/assets/1a5c1c7f-8b6f-460b-9d82-b8b4b9e53a56">